meshtastic/node_runtime/seturl/__init__.py

```
@@ -0,0 +1,36 @@
+    "_SetUrlAddOnlyPlan",
+    "_SetUrlReplacePlan",
+    "_SetUrlAddOnlyExecutionState",
+    "_SetUrlReplaceExecutionState",
+    "_SetUrlParser",
+    "_SetUrlAddOnlyPlanner",
+    "_SetUrlReplacePlanner",
+    "_SetUrlCacheManager",
+    "_SetUrlExecutionEngine",
+    "_SetUrlRollbackEngine",
+    "_SetUrlTransactionCoordinator",
+]
```

Ruff (0.15.7)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)